app/components/rank-step.jsx

```diff
@@ -0,0 +1,67 @@
+//https://github.com/EnCiv/civil-pursuit/issues/65
+import React from 'react'
+import { createUseStyles } from 'react-jss'
+import cx from 'classnames'
+import Point from './point.jsx'
+import PointGroup from './point-group.jsx'
+
+import Ranking from './util/ranking.jsx'
+
+function RankStep(props) {
+  const classes = useStylesFromThemeFunction()
+  const {
+    className = '', // may or may not be passed. Should be applied to the outer most tag, after local classNames
+    onDone = undefined, // a function that is called when the button is clicked.  - if it exists
+    children,
+    active = true,
+    pointList = [],
+    rankList = [],
+    ...otherProps
+  } = props
+  const setRank = (index, val) => {
+    for (let i = 0; i < rankList.length; i++) {
+      if (rankList[i].id === index) {
+        rankList[i]['rank'] = val
+        break
+      }
+    }
+  }
+  const findRank = index => {
+    for (let i = 0; i < rankList.length; i++) {
+      if (rankList[i].id === index) {
+        return i
+      }
+    }
+  }
+  return (
+    <div className={cx(classes.wrapper, className)} {...otherProps}>
+      <div className={classes.pointDiv}>
+        {pointList.map((point, i) => (
+          <Point key={point._id} subject={point.subject}>
+            {rankList[i] && (
+              <Ranking
+                defaultValue={rankList[findRank(point._id)]['rank']}
+                onDone={({ valid, value }) => {
+                  setRank(point._id, value)
+                }}
```

Comment:
if all of the ranks are set to something, this component should call onDone with {valid:true, value: 1).  If not all are set, then it should call onDone({valid: false, value: percent done})

Also we need have logic here that tests if there are 2 most important and 1 least important, and return valid: false in that case.

Also, I want to point out a react issues - after you call setRank within this function, the rankList won't or can't be counted on to be updated until the next render.  So you need to do the tests above, without depending on setRank.